source/cellular_3gpp_api.c

```diff
@@ -489,7 +489,7 @@ static CellularPktStatus_t _parseTimeZoneInfo( char * pTimeZoneResp,
 
     if( pktStatus == CELLULAR_PKT_STATUS_OK )
     {
-        LogDebug( ( "TimeZoneInfo: Timezone %d Year %d Month %d day %d,", pTimeInfo->timeZone,
+        LogDebug( ( "TimeZoneInfo: Timezone %ld Year %d Month %d day %d,", pTimeInfo->timeZone,
```

Comment:
year is `uint16_t year;` are you getting warning for this as well? %u should suffice here